lib/pronto/config.rb

```diff
@@ -11,6 +11,10 @@ def initialize(config_hash = ConfigFile.new.to_h)
       end
     end
 
+    def consolidate_comments?
+      !@config_hash['consolidate_comments'].nil?
```

Comment:
@doomspork @jhass having consistency between config yml and ENV config would be amazing. I'd gladly accept a PR that improves it.